File: agent/core/logger.py
import sys
from datetime import datetime

def info(tag, message):
    timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
    print(f"[{timestamp}] [{tag}] {message}")
    sys.stdout.flush()

def error(tag, message):
    timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
    print(f"[{timestamp}] [{tag}] ERROR: {message}", file=sys.stderr)
    sys.stderr.flush()

File: go_agent/logger/logger.go
package logger

import (
	"fmt"
	"os"
	"time"
)

func now() string {
	return time.Now().Format("2006-01-02 15:04:05")
}

func Info(tag, message string) {
	fmt.Printf("[%s] [%s] %s\n", now(), tag, message)
}

func Error(tag, message string) {
	fmt.Fprintf(os.Stderr, "[%s] [%s] ERROR: %s\n", now(), tag, message)
}
